src/config.json

```diff
@@ -2,5 +2,8 @@
   "axios": {
     "baseURL": "http://localhost:3001",
     "withCredentials": true
+  },
+  "opengraph": {
+    "basePageImageURL": "http://localhost:3000/favicon.ico"
```

Comment:
Use the baseUrl configuration and add favicon hardcoded.

Reply:
removed